variables.tf

```diff
@@ -112,7 +112,13 @@ variable "must_change_password" {
 }
 
 variable "grant_default_roles" {
-  description = "Whether to grant default_role and default_secondary_roles to Snowflake User"
+  description = "Whether to grant default_role to Snowflake User"
+  type        = bool
+  default     = true
+}
+
+variable "enforce_defaults" {
```

Comment:
perhaps `ignore_changes_on_defaults` is more descriptive?

Reply:
It was my orginal idea, but `enforce_defaults` was just shorter... But I can rollback